### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by Seg-Faulted** [3Shain/dxmt#14] [Q&A] [answered]
Hey, I'm new to Wine without a graphical client, and I saw this when I was looking for D3dmetal instead of DXVK, and saw on forums that it performs better than D3dmetal. Given that there were no installation instructions posted, I'm guessing people who use wine base often are installing this. I have the artifact downloaded from the workflow, I was just wondering what to do now?

**Comment by 3Shain** [comment, marked as answer]
Hi, DXMT is not ready for end-user at the moment. Only certain games are supported by dedicated project  that packaging their own wine with DXMT. Unless you want to develop, you don't need to do anything but wait for the release (which won't be far away).
